Fix typo in default database quota definition

pgorszkowski-igalia opened the pull request:
#### 9603260a265d713bb75d03de69e634209ec2f0d5
<pre>
Fix typo in default database quota definition
<a href="https://bugs.webkit.org/show_bug.cgi?id=254512">https://bugs.webkit.org/show_bug.cgi?id=254512</a>

Reviewed by Michael Catanzaro.

The current definition of the default database quota is wrongly
specified.

The fix uses MB constexpr to make it more obvious.

* Source/WebKit/UIProcess/API/glib/WebKitUIClient.cpp:

Canonical link: <a href="https://commits.webkit.org/262493@main">https://commits.webkit.org/262493@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a5f4c8fcb936a999df0dfed49ff366abe52f4ad1

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/888 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/912 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/947 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/1260 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/784 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/962 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/997 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/994 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/896 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/833 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/835 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/1195 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/890 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/830 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/830 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/803 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/856 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/1854 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/844 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/1240 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/791 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/841 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/414 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/858 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

mcatanzaro commented on Source/WebKit/UIProcess/API/glib/WebKitUIClient.cpp:
```diff
@@ -264,7 +264,7 @@ class UIClient : public API::UIClient {
     void exceededDatabaseQuota(WebPageProxy*, WebFrameProxy*, API::SecurityOrigin*, const String&, const String&, unsigned long long /*currentQuota*/, unsigned long long /*currentOriginUsage*/, unsigned long long /*currentDatabaseUsage*/, unsigned long long /*expectedUsage*/, Function<void(unsigned long long)>&& completionHandler) final
     {
-        static const unsigned long long defaultQuota = 5 * 1024 * 1204; // 5 MB
+        static const unsigned long long defaultQuota = 5 * MB;
```
Ah, I see 1204 instead of 1024. I would say "typo" rather than "lapse."

So this reduces the quota from 5 * 1024 * 1204 bytes down to 5 * 1000 * 1000 bytes. Sure.

mcatanzaro replied:
Oh no, I see we've incorrectly defined MB as 1024 * 1024 in wtf/StdLibExtras.h. Oh boy....

mcatanzaro replied:
And it seems I had already discovered and reported a bug for this: https://bugs.webkit.org/show_bug.cgi?id=209314

mcatanzaro reviewed (approved):
"wronly" -> "wrongly"

pgorszkowski-igalia committed:
Fix typo in default database quota definition
https://bugs.webkit.org/show_bug.cgi?id=254512

Reviewed by Michael Catanzaro.

The current definition of the default database quota is wrongly
specified.

The fix uses MB constexpr to make it more obvious.

* Source/WebKit/UIProcess/API/glib/WebKitUIClient.cpp:

Canonical link: https://commits.webkit.org/262493@main
